src/bit-components.js

```diff
@@ -167,9 +167,14 @@ export const SceneLoader = defineComponent({ src: Types.ui32 });
 SceneLoader.src[$isStringType] = true;
 
 export const MediaImage = defineComponent({
-  cacheKey: Types.ui32
+  cacheKey: Types.ui32,
+  projection: Types.ui8,
+  alphaMode: Types.ui8,
+  alphaCutoff: Types.f32
 });
 MediaImage.cacheKey[$isStringType] = true;
+MediaImage.projection[$isStringType] = true;
+MediaImage.alphaMode[$isStringType] = true;
```

Comment:
If I'm right, `projection` and `alphaMode` are not really needed to be strings in the component, aren't they? They seem to be used just as type (enum).

Reply:
The enum is string based I think to make it easier to handle comparisons as those parameters as string in the glTFs.

Reply:
Can you elaborate this for me a bit more, please?

> handle comparisons as those parameters as string in the glTFs

What I though is, if I'm right, `projection` and `alphaMode` are finally used just for comparing like `projection === ProjectionMode.SPHERE_EQUIRECTANGULAR` or `switch(alphaMode) { case AlphaMode.Opaqueue: ...` in Hubs Client. So I don't think the enum values in Hubs Client really need to be strings. Instead they can be just integer and component data type can be simply `ui8`. Strings doesn't really fit to bitECS API and we use hack. Probably it would be good to avoid strings in component data where possible.

But this is kind of optimization. We may think of in another PR.

By the way, if you want to use strings, better to use `ui32` component data type. `ui8` can be easily shortage.

Reply:
Good call, I didn't pay much attention to it and I'm not sure why we decided to make them strings early on but I've migrated them to numbers to simplify components.